.semaphore/semaphore.yml

```diff
@@ -108,6 +104,10 @@ blocks:
     minutes: 30
   dependencies: []
   task:
+    agent:
```

Comment:
The apiserver job seems to be timing out mid-build a fair bit recently. Not sure if this will fix it but hoping that it does.